.github/workflows/release.yaml

```diff
@@ -15,6 +15,10 @@ jobs:
         run: |
           VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/v##g")
           echo ::set-output name=version::${VERSION}
+          MINOR_VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/v##g" | grep -o -E "^[0-9]+\.[0-9]+")
+          echo ::set-output name=minor_version::${MINOR_VERSION}
+          LATEST_VERSION=$(curl -s -u "${{ github.repository_owner }}:${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/scalar-labs/scalardb/tags | jq -r .[].name | sort --version-sort -r | head -n 1 | sed -e "s#v##g")
+          echo ::set-output name=latest_version::${LATEST_VERSION}
```

Comment:
In these steps, I use GitHub API to get the information of `latest version` of Scalar DB.
You can try it by following command in your local.
```console
curl -s https://api.github.com/repos/scalar-labs/scalardb/tags | jq -r .[].name | sort --version-sort -r | head -n 1 | sed -e "s#v##g"
```
For example, we can get the string `3.5.2` now.